src/test/java/org/fundacionjala/coding/maria/Kata8Test.java

```diff
@@ -0,0 +1,71 @@
+package org.fundacionjala.coding.maria;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Kata 8.
+ */
+public class Kata8Test {
+    private Kata8 kata8;
+    private static String COMMONLINE1 =  " _  _     _  _        _  _ ";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "COMMONLINE1" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)